lib/ui/screens/event_screen.dart

```diff
@@ -118,6 +118,41 @@ class _EventScreenState extends State<EventScreen> {
   /// Create the title, start, end, location and price of an event.
   Widget _makeBasicEventInfo(Event event) {
     final textTheme = Theme.of(context).textTheme;
+
+    List<Widget> clickableOrganisers = [];
+    bool commaFlag = false;
+
+    for (ListGroup org in event.organisers) {
+      if (commaFlag) {
+        clickableOrganisers.add(Text(', ', style: textTheme.titleSmall));
+      }
+
+      bool spaceFlag = false;
+      List<String> nameParts = org.name.split(' ');
+
+      for (String namePart in nameParts) {
+        if (spaceFlag) {
+          clickableOrganisers.add(GestureDetector(
+              onTap: () {
+                context
+                    .pushNamed('group', params: {'groupPk': org.pk.toString()});
+              },
+              child: Text(' ', style: textTheme.titleSmall)));
+        }
+
+        clickableOrganisers.add(GestureDetector(
+            onTap: () {
+              context
+                  .pushNamed('group', params: {'groupPk': org.pk.toString()});
+            },
+            child: Text(namePart, style: textTheme.titleSmall)));
+
+        spaceFlag = true;
+      }
```

Comment:
Any reason you do not first create the string and then use one `GestureDetector`?

Reply:
This was my solution at first, but this (from what I could tell) did not allow me to wrap names like "Board 2020-2021" onto the new line, as the whole text would be inside a singular instance of `GestureDetector`.

Reply:
You mean something like 
```
{lots of text} board
2020-2021
```
? I think it would make sense to have the groupnames be split in between groups and not within a single group. Do you have any examples of where it would result in odd results? (besides extremely long groupnames, I think we can ignore those).

Reply:
With wrapping:
![Screenshot_20230427-135009](https://user-images.githubusercontent.com/24889701/234854335-b92a7ef8-3925-4dea-ad92-df1a8ea681a0.png)

Without:
![Screenshot_20230427-135335](https://user-images.githubusercontent.com/24889701/234854429-eb630229-a543-4e02-9b24-59460d30cf7b.png)

I just think the first looks cleaner

Reply:
Hmm, yeah it may be a bit of an edgecase but it does look better. Maybe putting it in a different function would be best

Reply:
Donezo!

Reply:
It might be more accurate to have it be a RichtText with spans (see terms and conditions)

Reply:
Ah yeah I think that would be better for accessibility as well